src/styles/application.css

```diff
@@ -99,7 +99,7 @@ main > [role=main].sto-myapps {
   }
 
   .sto-modal-page-app {
-    height: calc(100vh - 6rem)
+    height: calc(100% - 6rem)
```

Comment:
I am pretty sure it will cause an regression in responsive mode.

Reply:
This was an issue actually. Safari and Safari iOS don't handle `vh` and `vw` in calc function, this is why we changed that here.

Reply:
Actually not sure we need this change anymore, since we used Portal. Worth checking.

Reply:
This fis seems not necessary anymore, so I removed the commit